frontend/src/components/QrCodeScanner.jsx

```
@@ -67,25 +69,47 @@ const QrCodeScanner = () => {
+            QR Code Data:
+            <button
+              onClick={() => copyToClipboard(qrData)}
+              className="ml-2 text-sm text-primary hover:underline"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                className="h-5 w-5 inline-block"
+                viewBox="0 0 20 20"
+                fill="currentColor"
+              >
                 <path d="M8 3a1 1 0 011-1h2a1 1 0 110 2H9a1 1 0 01-1-1z" />
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)